docs/usage.md

```diff
@@ -119,3 +119,34 @@ ambassador.container.image=replacement image name here
 vncrecorder.container.image=replacement image name here
 tinyimage.container.image=replacement image name here
 ```
+
+## JUnit
+
+### Junit 4
+ 
+**JUnit4 @Rule/@ClassRule**: this mode starts container before your tests and tears it down afterwards.
+
+Add a @Rule or @ClassRule to your test class, e.g.:
+
+```java
+public class SimpleMySQLTest {
+    @Rule
+    public MySQLContainer mysql = new MySQLContainer();
+```
+
+### JUnit 5
+
+**JUnit5**: you need to manually start container in *@BeforeAll* annotated method in test (tear down happens automatically on test end)
```

Comment:
this example also does not reflect the previous one, with JUnit 4. Maybe we should replace it with `@BeforeEach`/`@AfterEach`

Reply:
Or alternatively, use `@ClassRule` in the JUnit4 example.

Reply:
Also please uppercase `You` and `.` at the end of the sentence.
`You need to manually start the container in a *@BeforeAll* annotated method in your test`

Reply:
Also the tear down happen on JVM exit.

Reply:
Agree, will adjust it